The source patch keyring_path_change.patch import
tsconfig, so add tsconfig to Depends

--- a/debian/control
+++ b/debian/control
@@ -20,6 +20,7 @@ Package: python3-keyring
 Architecture: all
 Depends: python3-jeepney (>= 0.4.2),
          python3-secretstorage (>= 3.2),
+         tsconfig
          ${misc:Depends},
          ${python3:Depends}
 Suggests: gnome-keyring, libkf5wallet-bin, python3-dbus, python3-keyrings.alt
